.README/rules/require-jsdoc.md

```diff
@@ -34,6 +34,11 @@ be checked by the rule.
   - `FunctionExpression`
   - `MethodDefinition`
 
+- `contexts` - Set this to a string or array of strings representing the additional
```

Comment:
This belongs to another pr.

Reply:
I can move it to another PR, but note that it all leverages the same code.